themes/unfold/unfold.theme

```diff
@@ -0,0 +1,39 @@
+
+_lp_unfold_theme_activate() {
+    _lp_default_theme_activate
+}
+
+_lp_unfold_theme_directory() {
+    _lp_default_theme_directory
+}
+
+_lp_unfold_theme_prompt() {
+    _lp_default_theme_prompt_data
+
+    if [[ -z "${LP_PS1-}" ]]; then
```

Comment:
Get rid of this check as well. Best case it will never happen, since we are not importing a template file to set it, worst case the user switched from the default theme, `$LP_PS1` is still set, and their prompt will never change.

The `else` block below can disappear then obviously.